### PR TITLE
feat(influxdb2): Add backup cronjob

### PR DESCRIPTION
**Pull request by abhi1693**
- Adds backup cronjob using [rclone](https://rclone.org/)
- Bumps chart to `2.0.2`

### CODE REVIEW

**Comment by abhi1693** [comment]
@rawkode @timhallinflux Kindly give this a try whenever you can :)

**Review by timhallinflux — APPROVED** [review]

**Comment by florianrusch** [comment]
@rawkode @timhallinflux @abhi1693 can we bring some progress on this? I really looking forward to it and would like to help out if I can!

**Comment by abhi1693** [comment]
@florianrusch I also submitted a similar PR at https://github.com/bitnami/charts/pull/8317 if you are interested

File: charts/influxdb2/templates/_helpers.tpl [hunk 1]
```diff
@@ -51,3 +51,14 @@ Selector labels
...
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "influxdb.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create -}}
+    {{ default (include "influxdb.fullname" .) .Values.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
```
Comment (florianrusch) [inline charts/influxdb2/templates/_helpers.tpl:64]: Why do you need it? You only use it in the cronjob. There is no SA resource anywhere or?

**Review by florianrusch — CHANGES_REQUESTED** [review]

**Comment by abhi1693** [comment]
Closing since maintainers are unresponsive

**Pull request closed without merging**